.requirements

```diff
@@ -13,3 +13,4 @@ k8s.io/code-generator/cmd/deepcopy-gen
 k8s.io/code-generator/cmd/defaulter-gen
 k8s.io/code-generator/cmd/informer-gen
 k8s.io/code-generator/cmd/lister-gen
+sigs.k8s.io/mdtoc
```

Comment:
probably not ideal here since it ends up in the build image but i don't think we have a better place to put it

Reply:
okay 😞